micro-ui/web/micro-ui-internals/packages/modules/hrms/CHANGELOG.md

```
@@ -1,3 +1,7 @@
+
+## [1.8.16] [25-July-2025] 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)